src/main/kotlin/com/mineinabyss/looty/ecs/components/LootyPack.kt

```diff
@@ -0,0 +1,11 @@
+package com.mineinabyss.looty.ecs.components
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+@SerialName("looty:resourcepack")
+class LootyPack(
+    val model: String = "",
```

Comment:
Because you wanna have the option to specify the texture and not require a modelfile.
Then looty will generate the modelfile  to tie the item to a texture, or you can just set to a model if u have a 3d item etc